source/uri-options/tests/tls-options.yml

```diff
@@ -71,19 +71,51 @@ tests:
         auth: ~
         options: {}
     -
-        description: "tlsInsecure=true and tlsAllowInvalidCertificates=false warns"
-        uri: "mongodb://example.com/?tlsInsecure=true&tlsAllowInvalidCertificates=false"
+        description: "tlsInsecure and tlsAllowInvalidCertificates both present raises an error"
```

Comment:
Let's be paranoid add two more tests, where the parameter values are false.

Reply:
Will do